### DISCUSSION TITLE
XAMPP user---Error: Backup failed

### DISCUSSION THREAD

**Discussion by huifei0129** [snipe/snipe-it#19374] [Q&A]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

**_Version v7.0.5 - build 14092 (master)
XAMPP user_** 

![image](https://github.com/snipe/snipe-it/assets/156650496/18e2877d-3b44-48f6-bc14-3d301ae6b4d6)
![image](https://github.com/snipe/snipe-it/assets/156650496/ff5be6de-6968-402d-a3c4-c9046fe0f6db)

**If I try it with the CLI command on the server itself, the backup gets generated.**
![image](https://github.com/snipe/snipe-it/assets/156650496/bbd57470-ba79-417c-81dd-63af6fc9ca5d)

**_But I get an error if I try to generate a backup through the GUI._** 
![image](https://github.com/snipe/snipe-it/assets/156650496/97e54f1a-89de-4773-9668-7034c4dbcc36)

**I have check the same issue #4862 #10234 #14593 #14910  #14891 #2454**
![image](https://github.com/snipe/snipe-it/assets/156650496/96d17d4f-cfdc-4804-abe7-8c6f2c213ee5)

**grant all permission**
![image](https://github.com/snipe/snipe-it/assets/156650496/49d918cf-5676-4eae-aee9-377540751caf)

### Reproduction steps

1.If I try it with the CLI command on the server itself, the backup gets generated.
2.I get an error if I try to generate a backup through the GUI. 
3.
...

### Expected behavior

help to solve this error

### Screenshots

Error: Backup failed because: The dump process failed with a none successful exitcode. Exitcode ======== 2: Misuse of shell builtins Output ====== <no output> Error Output ============ mysqldump: Got error: 2004: "Can't create TCP/IP socket (10106)" when trying to connect . #0 C:\xampp\htdocs\snipe-it\vendor\spatie\db-dumper\src\DbDumper.php(194): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process)) #1 C:\xampp\htdocs\snipe-it\vendor\spatie\db-dumper\src\Databases\MySql.php(161):  Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul(Object(Symfony\Component\Process\Process), 'C:\\xampp\\htdo

### Snipe-IT Version

Version v7.0.5 - build 14092 (master)

### Operating System

Windows

### Web Server

Apache，XAMPP，mysql

### PHP Version

8.1.25

**Comment by ufukozer** [comment]
Hello,

I am using XAMPP. I can take manual backup with php artisan snipeit:backup. I get an error when I try to take backup in GUI.

I edited the DB_DUMP_PATH path in the .env file, but I don't understand the user authorization part. Snipe-IT is running on a server in my domain. As local user only administrator account is enabled and has full control. But I can't take a backup on the GUI, the error persists.  Can you help me, I could not find any solution.

![image](https://github.com/user-attachments/assets/23387bf4-ab0d-4436-bc29-c5b365f5692e)

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by ufukozer** [comment]
Hello, unfortunately we could not find a solution. Snipe-it version: 8.2.1
